lib/features/base/reloadable/reloadable_controller.dart

```diff
@@ -154,10 +155,11 @@ abstract class ReloadableController extends BaseController {
         _deleteCredentialInteractor.execute(),
         _cachingManager.clearAll(),
         _languageCacheManager.removeLanguage(),
+        _cachingManager.closeHive(),
       ]);
       _authorizationInterceptors.clear();
       _authorizationIsolateInterceptors.clear();
-      await HiveCacheConfig().closeHive();
+      _cachingManager.deleteFcmToken();
```

Comment:
IMO, call directly to `FCM Receiver` is ok for me. It is perfect if you create an interface for `push_receiver`. But it is still ok without it